app/src/main/java/practicum/getfitla_v3/NutritionDetailActivity.java

```diff
@@ -1,34 +1,53 @@
 package practicum.getfitla_v3;
 
-import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
-import android.view.View;
+import android.widget.TextView;
+
 //Please ignore this page for now, this is a dummy page
-public class NutritionDetailActivity extends AppCompatActivity{
+public class NutritionDetailActivity extends AppCompatActivity {
+    private int passedid;
+    private String passedname;
+    private String passedshortdesc;
+    private String passedprice;
+    private int passedimage;
+    private String passedprepTime;
+    private String passedservingSize;
+    private String passedcalories;
+    private String passedequipment;
+    private String passedprocess;
+    private String passedingredients;
 
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.individual_nutrition_detail);
-    }
-
-    public void onClick_Maps(View view) {
-        Intent intent = new Intent(this, MapsActivity.class);
-        startActivity(intent);
 
+        //unpack the bundled information from previous activity
+        Bundle data = getIntent().getExtras();
+
+
+        NutritionItemFormat CurItem = (NutritionItemFormat) data.getParcelable("RecipeInfo");
+        passedid = CurItem.getId();
+        passedname = CurItem.getName();
```

Comment:
you only need passedname and passedprocess to set up your text view below, you can delete everything else.

Reply:
@Danhuth Can u give me the equivalent NUTRITION_SHEET_URL